x-pack/plugins/fleet/server/services/secrets.test.ts

```diff
@@ -968,8 +968,8 @@ describe('secrets', () => {
       type: 'integration',
       status: 'not_installed',
       vars: [
-        { name: 'pkg-secret-1', type: 'text', secret: true, required: true },
-        { name: 'pkg-secret-2', type: 'text', secret: true, required: false },
+        { name: 'pkg-secret-1', type: 'text', secret: true, required: false },
+        { name: 'pkg-secret-2', type: 'text', secret: true, required: true },
```

Comment:
is it expected to change the required flags here? the test in line 1202 doesn't populate a required secret value.

Reply:
This is sort of leftover from me trying to see if there was an issue with the order of secrets - so I was just moving the required secret to the second element as an experiment. It doesn't the impact the behavior AFAIK so it could probably be reverted.